internals/daemon/api_exec_test.go

```diff
@@ -108,6 +108,28 @@ func (s *execSuite) TestEnvironment(c *C) {
 	c.Check(stderr, Equals, "")
 }
 
+func (s *execSuite) TestEnvironmentInheritedFromDaemon(c *C) {
+	oldEnv, envWasSet := os.LookupEnv("FOO")
+	err := os.Setenv("FOO", "bar")
```

Comment:
is this safe? other tests simply set the env var at the test execution time

Reply:
It should be safe. The `Environment` field other tests are using in this file seems to be the client env.

Reply:
I guess it's not thread-safe or rather goroutine-safe, but we do this with `os.Args` already.

Reply:
However, I'd put this into a new `fakeEnv()` function. It's likely other tests may want it too in future.

Reply:
Added in https://github.com/canonical/pebble/pull/234/commits/8edd665543744e57047b848640031424bbb0a25e. Thank you.